data-processing-lib/python/src/data_processing/runtime/transform_file_processor.py

```diff
@@ -21,17 +23,26 @@ class AbstractTransformFileProcessor:
     This is the the base class implementing processing of a single binary file
     """
 
-    def __init__(self):
+    def __init__(
+            self,
+            data_access_factory: DataAccessFactoryBase,
+            transform_parameters: dict[str, Any],
+    ):
         """
         Init method
+        :param data_access_factory: Data Access Factory
+        :param transform_parameters: Transform parameters
         """
-        self.data_access = None
```

Comment:
As long as we're moving this around, can we name it what it is, that is data_access_factory?  Same for the key. it should be "data_access_factory", although for backwards compatibility, we need to keep "data_access".

Reply:
But its really data access:
```
self.data_access = data_access_factory.create_data_access()
```

Reply:
ah good. i stand corrected.